### ISSUE TITLE
Chefkoch integration broken

### ISSUE THREAD

**Issue by rixx** [AberDerBart/recipemd-extract#14]
All tries return

```
In plugin " .chefkoch ": Error parsing recipe: 'NoneType' object has no attribute 'text'
```

**Comment by AberDerBart** [comment]
The chefkoch plugin is not actively used anymore. Chefkoch supports recipe schema, so chefkoch recipes are parsed by that parser. It was left in the codebase to serve as a fallback should chefkoch stop supporting recipe schema.

Do you want to use recipemd-extract as a library? I then suggest using scrape_schema_recipe directly, as it will be more stable than recipemd-extract. Would that solve your issue? I think it is more futureproof to remove the chefkoch-specific plugin then.

**Comment by rixx** [comment]
I installed `recipemd-extract` as in the instructions and ran `recipemd-extract <chefkoch-url>`, which led to me receiving the above mentioned error. Since I wanted to import some chefkoch recipes, I fixed the problem as it presented itself, and thought I'd share.

I don't know the internals of this tools, so I can't tell you *why* the plugin got used, but it sure did. Completely understand that you don't want to merge theoretically unused code, so I'll close the PR, and use the fork. Thanks for your reply!
